recipes/pysocks/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{%set name = "pysocks" %}
+{%set camelName = "PySocks" %}
+{%set version = "1.5.7" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "e51c7694b10288e6fd9a28e15c0bcce9aca0327e7b32ebcd9af05fcd56f38b88" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ camelName }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - socks
+
+about:
+  home: https://github.com/Anorov/PySocks
+  license: BSD 3-Clause
```

Comment:
Ok, please just add a comment in the recipe with a reference to that issue for now.